contrib/guix/manifest.scm

```diff
@@ -15,8 +15,10 @@
              (gnu packages ninja)
              (gnu packages pkg-config)
              ((gnu packages python) #:select (python-minimal))
-             ((gnu packages python-build) #:select (python-tomli python-poetry-core))
+             ((gnu packages python-build) #:select (python-poetry-core))
              ((gnu packages python-crypto) #:select (python-asn1crypto))
+             ((gnu packages python-science) #:select (python-scikit-build-core))
+             ((gnu packages python-xyz) #:select (python-pydantic-2 python-pydantic-core))
```

Comment:
765922d8022d3addeb80b5f3f6e041f7fd2ad3ed: do you know why https://codeberg.org/guix/guix/pulls/353/files only needs `python-pydantic-2` and not `python-pydantic-core`?